website/docs/testing/markdown/doc-components.md

```diff
@@ -0,0 +1,120 @@
+---
+title: DOC Components
+---
+
+## Doc::Badge
+
+<Doc::Badge @type="neutral">Spacebar</Doc::Badge>
+
+## Doc::ColorSwatch
+
+<!-- algolia-ignore-start -->
+<div>
+  {{#each this.colors.semantic.focus as |color|}}
+    <Doc::ColorSwatch @color={{color}} />
+  {{else}}
+    <p>No tokens found for “semantic/focus” colors.</p>
+  {{/each}}
+</div>
+<!-- algolia-ignore-end -->
+
```

Comment:
This is a great point, because it is useful in testing and ensuring quality. The reason I have it wrapped in a div is because the template syntax breaks Showdown transformation if it is not present.
If you look at how we use it website/docs/foundations/colors/partials/other/palette.md it is being wrapped in a div like this, most likely for the same reason.

Reply:
Yes, likely same reason.